pages/common/hyperfine.md

```diff
@@ -0,0 +1,23 @@
+# hyperfine
+
+> A command-line benchmarking tool.
+
+- Run a basic benchmark:
+
+`hyperine '{{make}}'`
```

Comment:
missing f?

Reply:
Indeed, sorry about that, editing now...